lib/promiscuous_black_hole/db.rb

```diff
@@ -16,6 +16,11 @@ def self.search_path
       fetch('show search_path').first[:search_path]
     end
 
+    def self.max_identifier_length
+      @max_identifier_length ||=
```

Comment:
this is not threadsafe, but we don't care because it'll always return the same thing